### DISCUSSION TITLE
How do I embed it in Shopify?

### DISCUSSION THREAD

**Discussion by StrongereCommerce** [code4recovery/12-step-meeting-list#889] [Q&A] [answered]
Our intergroup uses Shopify as they also sell literature. We have a google sheets doc with all meetings and details. How can I utilize this great tool to help fellows find out meetings easier?

**Comment by 97eighty** [comment, marked as answer]
@SwissMediaRocks 

Create a custom page in Shopify and use the TSML UI to connect your data source. Check out https://tsml-ui.code4recovery.org for an example.

**Reply by StrongereCommerce** [reply]
Thanks for the quick help!!
